Dockerfile

```diff
@@ -2,19 +2,22 @@ FROM nfcore/base:1.14
 LABEL authors="Leon Bichmann" \
       description="Docker image containing all software requirements for the nf-core/diaproteomics pipeline"
 
+# Install mamba
+RUN conda install mamba -n base -c conda-forge
+
 # Install the conda environment
 COPY environment.yml /
-RUN conda env create --quiet -f /environment.yml && conda clean -a
+RUN mamba env create --quiet -f /environment.yml && mamba clean -a
 
 # Add conda installation dir to PATH (instead of doing 'conda activate')
 ENV PATH /opt/conda/envs/nf-core-diaproteomics-1.2.5dev/bin:$PATH
```

Comment:
does the path need to be changed to mamba as well?

Reply:
I wondered that but I don't think so.

Just checked with `docker run -it nfcore/diaproteomics:dev sh` and the path seems fine 👍🏻

Reply:
just tested it and had a problem with running `pyprophet --version` due to a missing library